Sources/Paywalls/Events/Networking/PaywallEventsRequest.swift

```diff
@@ -16,20 +16,39 @@ import Foundation
 /// The content of a request to the events endpoints.
 struct PaywallEventsRequest {
```

Comment:
Thought about renaming this, but we are still using the `api-paywalls` to post all type of events. So I am not sure. I thought about renaming to `FeatureEventsRequest`

Reply:
Hmm I think we should rename it or it would be very confusing down the line, and maybe just keep using the old endpoint for now, (or even alias to a new endpoint name in the backend and deprecate the old one... But that can come in a separate PR.